applications/asset_tracker/src/cloud_codec/cloud_codec.c

```diff
@@ -18,11 +19,19 @@
 #include <logging/log.h>
 #include "env_sensors.h"
 
+LOG_MODULE_REGISTER(cloud_codec, CONFIG_NRF_CLOUD_LOG_LEVEL);
```

Comment:
i see that the other asset tracker files use printk instead of the logging module.
what is the reason for that?
should the log messages i added just be changed to printk?

Reply:
All, or at least almost all, samples in NCS use printk, and it's sort of a standard way of doing things. I think the rationale was that a sample should output information even though the logging subsystem is disabled. When the asset tracker went to being considered an application instead of a sample, we should probably have moved to the logger, as a real-world application would probably do that.

Reply:
i changed the three LOG_ items to printk... they can be updated to LOG_ for TG91-137.